test/unit/mdc-slider/foundation.test.js

```diff
@@ -426,3 +427,57 @@ test('#setMin updates "aria-valuemin" to the new minimum', () => {
 
   raf.restore();
 });
+
+test('#isDisabled/#setDisabled retrieves / sets the disabled state, respectively', () => {
+  const {foundation} = setupTest();
+  foundation.setDisabled(true);
+
+  assert.isTrue(foundation.isDisabled());
+});
+
+test('#setDisabled adds the mdc-slider--disabled class when given true', () => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.setDisabled(true);
+
+  td.verify(mockAdapter.addClass(cssClasses.DISABLED));
+});
+
+test('#setDisabled adds "aria-disabled=true" when given true', () => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.setDisabled(true);
+
+  td.verify(mockAdapter.setThumbAttribute('aria-disabled', 'true'));
+});
+
+test('#setDisabled removes the tabindex attribute when given true', () => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.setDisabled(true);
+
+  td.verify(mockAdapter.removeThumbAttribute('tabindex'));
+});
+
+test('#setDisabled removes the mdc-slider--disabled class when given false', () => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.setDisabled(false);
+
+  td.verify(mockAdapter.removeClass(cssClasses.DISABLED));
+});
+
+test('#setDisabled removes the "aria-disabled" attribute when given false', () => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.setDisabled(false);
+
+  td.verify(mockAdapter.removeThumbAttribute('aria-disabled'));
```

Comment:
Is the string `"aria-disabled"` included in your constants? If so, you should use that string here in your verification instead of writing it yourself.

Reply:
Done!